datastream-cloud-storage/src/main/java/com/linkedin/datastream/cloud/storage/committer/GCSObjectCommitter.java

```diff
@@ -114,10 +117,40 @@ private static String getObjectName(final String destination,
                                         final long startOffset,
                                         final long endOffset,
                                         final String suffix,
-                                        final String fileExt) {
+                                        final String fileExt,
+                                        boolean isScribeParquetFileStructure) {
         String prefix = destination.substring(destination.indexOf("/") + 1);
 
-        return new StringBuilder()
+      // isScribeParquetFileStructure flag is used to indicate whether to follow scribe 2.0 file structure or the regular one.
+
+      // Scribe 2.0 Kafka topic name format: domain_name-event_name
+      // Eg: kafka topic: scribe_internal-healthcheck_evaluated, corresponding domain name: scribe_internal and event name: healthcheck_evaluated
+
+      // scribe parquet file structure: events/event_name/eventdate=2020-12-21/scribeKafkatopic+partition+startOffset+endOffset+suffix.parquet
+      // Eg: events/healthcheck_evaluated/eventdate=2021-02-22/scribe_internal-healthcheck_evaluated+0+187535+187631+1613970121085.parquet
+      if (isScribeParquetFileStructure) {
```

Comment:
Users may have different object name requirements. We should work on making this a dynamic datastream config parameter.